test/parallel/test-error-serdes-recursive-fast.js

```diff
@@ -0,0 +1,29 @@
+// Flags: --expose-internals --stack-size=128
```

Comment:
Are these just three tests testing the same thing but with different stack sizes?

Reply:
Yep, these belong to https://github.com/nodejs/node/pull/52674/commits/af75425ed3402ce1dffc736df7f27f40263b24c8 just to see if there's any difference in CI results on different platforms.
That commit also extracted the stack exhausting parts (primary suspect of flakiness) from the rest of the test to confirm that cyclic errors are the cause of timeout.